content/soarian/dstu2/care-provision/careplan.md

```diff
@@ -55,11 +55,11 @@ _Implementation Notes_
 Notes:
 
 * See details regarding special [date][date-parameter] requirements.
-* If the date parameter is null in the query, the API will return qualifying records with dates less than or equal to the date of the query as limited by the lesser of 1000 records or 30 days inclusive of the query date.
-* If a single date parameter is used in the query, the response will include qualifying records dated that day as limited by 1000 records.
-* If a valid date range is used in the query, the API will return qualifying records within the dates specified as limited by the lesser of 1000 records or 30 days starting at the upper limit of the range.
-* If a date range greater than 30 days is used in the query, the response will include qualifying records within the dates specified as limited by the lesser of 1000 records or 30 days starting at the upper limit of the range. In this case, the informational response in the narrative extension will indicate that the date range is greater than 30 days and should be refined.
+* If the date parameter is null in the query, the API will return qualifying records.
```

Comment:
To be consistent with other changes: "...the API will return *all* qualifying records."

Reply:
https://github.com/cerner/fhir.cerner.com/pull/187/commits/8aee949bc8c2cf0964f281a84c81ffa5b08fdfa2